cesm/flux_atmocn/shr_flux_mod.F90

```diff
@@ -257,22 +263,21 @@ SUBROUTINE flux_atmOcn(logunit, nMax  ,zbot  ,ubot  ,vbot  ,thbot ,   &
     real(R8)    :: tdiff(nMax)               ! tbot - ts
     real(R8)    :: vscl
 
+    real(R8)    :: ugust      ! function: gustiness as a function of convective rainfall.  
+    real(R8)    :: gprec   ! dummy arg ~ ?
 
     qsat(Tk)   = 640380.0_R8 / exp(5107.4_R8/Tk)
-
-    ! Large and Yeager 2009
-    cdn(Umps)  =  0.0027_R8 / min(33.0000_R8,Umps) + 0.000142_R8 + &
-         0.0000764_R8 * min(33.0000_R8,Umps) - 3.14807e-13_r8 * min(33.0000_R8,Umps)**6
-    ! Capped Large and Pond by wind
-    !   cdn(Umps)  =   0.0027_R8 / min(30.0_R8,Umps) + 0.000142_R8 + 0.0000764_R8 * min(30.0_R8,Umps) 
-    ! Capped Large and Pond by Cd
-    !   cdn(Umps) = min(0.0025_R8, (0.0027_R8 / Umps + 0.000142_R8 + 0.0000764_R8 * Umps ))
-    ! Large and Pond
-    !   cdn(Umps)  =   0.0027_R8 / Umps + 0.000142_R8 + 0.0000764_R8 * Umps
-
+    cdn(Umps)  =   0.0027_R8 / Umps + 0.000142_R8 + 0.0000764_R8 * Umps
```

Comment:
You removed the cap on this equation so this will affect not just cam_dev but all configurations?

Reply:
Thanks Jim, fixed this now; not something I intended to change.